resources/scss/componets/01_basics.scss

```diff
@@ -0,0 +1,131 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+// colors variable start
+
+$black: #000000;
+$white: #ffffff;
+$theme-color: #F51963;
+$theme-color-20: rgba(245, 25, 99, 0.2);
+$border-color: lightgray;
+$linear-gradient-theme: linear-gradient(45deg, #F51963 0%, #FBAE20 100%);
+$background-theme-color: #FAF9FF;
+$water-color: rgba(255, 255, 255, 0.1);
+
+// colors variable end
+
+// global start
+
+.background-theme-color{
+    background-color: $background-theme-color !important;
+}
```

Comment:
Move this scss into **background.scss** and remove this from this file please

Reply:
i remove this scss into background.scss